apps/server/src/utils/oscArgParser.ts

```diff
@@ -19,6 +19,16 @@ export function stringToOSCArgs(argsString: string | undefined): Argument[] {
       return { type: 's', value: argString.substring(1, argString.length - 1) };
     }
 
+    if (argString === 'TRUE') {
+      // NOTE: Boolean true
+      return { type: 'T', value: true };
+    }
+
+    if (argString === 'FALSE') {
+      // NOTE: Boolean false
+      return { type: 'F', value: false };
```

Comment:
Does this actually work? The `node-osc` library [does not support those new types](https://github.com/MylesBorins/node-osc/blob/e6ab7013fd7de599a07c60920aab0749aff0a837/lib/Message.mjs#L1) as far as I can see.

Reply:
Tested locally and it does send as boolean types.

Reply:
Thanks for checking 
Do you have any comments for the tests?

Reply:
Nope looks pretty complete (or at least the cases I could think of). Just change the contents of `describe` at the beginning looks like it wasn't changed from copy/paste.